exporter/exporter.go

```diff
@@ -95,8 +96,71 @@ func EcsClient() (client *ecs.Client) {
 	return client
 }
 
+func NewRedisClient() (client *r_kvstore.Client) {
```

Comment:
Please use https://golang.org/doc/effective_go.html#embedding to do some refactoring on this part.

We can create a base struct 

```go
type Client struct {
    Config alicloud.Config
}

type EcsClient struct {
    *Client
}

type RedisClient struct {
    *Client
}
```